modules/P03_firmware_virustotal_check.sh

```diff
@@ -30,7 +36,7 @@ P03_firmware_virustotal_check() {
 
     # based on code from vt-scan: https://github.com/sevsec/vt-scan
     local FSIZE
-    FSIZE=$(stat "$FIRMWARE_PATH" | grep "Size:" | awk '{print $2}')
+    FSIZE=$(stat "$FIRMWARE_PATH" | grep "Size:\|Größe:" | awk '{print $2}')
```

Comment:
Maybe we could use a language independent approach here:
```bash
stat -c %s ./DIR300_RevB_Firmware_216WWB01.bin
3645580
```

Reply:
master of bash ...